tests/PerformanceTests/Microsoft.Identity.Web.Perf.Client/TestRunner.cs

```diff
@@ -129,59 +193,50 @@ public async Task Run()
                         exceptionsDuringRun.AppendLine($"Exception in TestRunner at {i} of {userEndIndex} - {userStartIndex}: {ex.Message}");
                         exceptionsDuringRun.AppendLine($"{ex}");
                     }
-
-                    Console.Title = $"[{userStartIndex} - {userEndIndex}] #: {i}, Loop: {loop}, " +
-                        $"Time: {(DateTime.Now - startOverall).TotalMinutes:0.00}, " +
-                        $"Req: {requestsCounter}, Cache: {tokenReturnedFromCache}: {fromCache}, " +
-                        $"AuthFail: {authRequestFailureCount}, Fail: {catchAllFailureCount}";
```

Comment:
this information is super helpful when watching the overall progress. Should find a way to summarize across all the running threads....